Libraries/LibWeb/Painting/StackingContext.cpp

```diff
@@ -139,6 +139,19 @@ void StackingContext::paint_descendants(PaintContext& context, Paintable const&
         if (child.is_positioned() && z_index.value_or(0) == 0)
             return IterationDecision::Continue;
 
+        // https://www.w3.org/TR/CSS22/zindex.html
```

Comment:
We always use the editor's draft of specifications; please use the spec text from that version.